### ISSUE TITLE
Generate & review validation reports: ucd_json

### ISSUE THREAD

**Issue by aturner** [ucldc/rikolti#516]
- **Step 2: Generate reports** [status: in progress --> ready for review]
  - [x] Generate validation reports (run `validate_by_mapper DAG`): https://7a8067cb-3b99-477e-a883-7e311175a9b4.c3.us-west-2.airflow.amazonaws.com/dags/validate_by_mapper_type/grid?dag_run_id=manual__2023-10-31T18%3A51%3A47%2B00%3A00
  - [x] Add a comment (in this issue) to the output URLs
  - [x] Download the validation reports locally; prepend the collection registry number to the filename
  - [x] Upload validation reports to the [Rikolti Data Validation](https://drive.google.com/drive/folders/1AgDPsz6oyDhsfgRuLyjomaXYgE1sPLNT) folder
- **Step 3: Review reports**  [status: ready for review --> review in progress]
  - [x] Review [sample collections validation reports](https://docs.google.com/spreadsheets/d/1XkBwi8jiuGgrWvQBqjkrbavSl13yyrQNyRyEb_CQ4z0/edit#gid=1143782640); leave notes in the spreadsheet
  - [x] Quickly (and not very deeply) scan through other validation reports (to identify if any new metadata fields surface)
  - [x] Synthesize notes at the [mapper level](https://docs.google.com/spreadsheets/d/1XkBwi8jiuGgrWvQBqjkrbavSl13yyrQNyRyEb_CQ4z0/edit#gid=1100474517)
  - [x] Share an update (and surface any anomalies) during a weekly check-in (Mondays at 3:00pm)
- **Step 4: Have data validation issues? Address & resolve validation issues (synthesized mapper-level):**
  - [ ] Create a new issue (“Validation results: Rikolti mapper name“) and report/request CIC fixes & investigations. Assign to CIC.
  - [ ] Add a comment to link to the new “validation results” issue.
  - [ ] Add a “blocked/paused” label to this issue. Include a status note in this issue title (e,g., HOLD for CIC fixes; HOLD for code review; etc.) 
  - [ ] Move this issue back to the “To Do” pile.
- [Re-generate & re-review (repeat steps 2-3 above, as needed.]
- **Step 5: Have acceptable validation reports?**
  - [x] Move the valid (accepted) reports to the “VALID” folder in google drive.
  - [x] Update the [mapper progress spreadsheet](https://docs.google.com/spreadsheets/d/1XkBwi8jiuGgrWvQBqjkrbavSl13yyrQNyRyEb_CQ4z0/edit#gid=1100474517); indicate that the mapper is accepted

**Comment by christinklez** [comment]
Blocked by: https://github.com/ucldc/rikolti/issues/610

Once unblocked, return to the specific job & task, and re-run from the task that experienced the error. (No need to re-run prior successful tasks.)

**Comment by christinklez** [comment]
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/8/validation/11-02-2023T00:59:42.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#8/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/3378/validation/11-02-2023T00:59:44.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#3378/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/3696/validation/11-02-2023T00:59:46.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#3696/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/5656/validation/11-02-2023T00:59:47.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#5656/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/8918/validation/11-02-2023T00:59:52.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#8918/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/9757/validation/11-02-2023T01:00:13.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#9757/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/10503/validation/11-02-2023T01:00:17.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#10503/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27202/validation/11-02-2023T01:01:33.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27202/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27275/validation/11-02-2023T01:01:38.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27275/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27306/validation/11-02-2023T01:01:58.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27306/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27307/validation/11-02-2023T01:02:06.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27307/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27548/validation/11-02-2023T01:02:24.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27548/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27685/validation/11-02-2023T01:02:26.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#27685/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28047/validation/11-02-2023T01:02:28.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#28047/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28048/validation/11-02-2023T01:02:38.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#28048/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28049/validation/11-02-2023T01:02:40.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#28049/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28050/validation/11-02-2023T01:03:27.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#28050/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28052/validation/11-02-2023T01:03:28.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#28052/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28053/validation/11-02-2023T01:03:29.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#28053/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28054/validation/11-02-2023T01:03:30.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#28054/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28055/validation/11-02-2023T01:03:31.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#28055/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28056/validation/11-02-2023T01:03:33.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#28056/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28057/validation/11-02-2023T01:03:34.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#28057/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28058/validation/11-02-2023T01:03:34.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#28058/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28059/validation/11-02-2023T01:03:36.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#28059/

- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28060/validation/11-02-2023T01:03:39.csv
- [2023-11-02, 01:03:39 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://rikolti-data.s3.us-west-2.amazonaws.com/index.html#28060/

**Comment by christinklez** [comment]
Reported validation issues here: https://github.com/ucldc/rikolti/issues/621

**Comment by christinklez** [comment]
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/8/vernacular_metadata_2023-12-20T20:05:17/mapped_metadata_2023-12-20T21:40:32/validation_2023-12-20T21:51:10.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#8/vernacular_metadata_2023-12-20T20:05:17/mapped_metadata_2023-12-20T21:40:32/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/3378/vernacular_metadata_2023-12-20T20:48:35/mapped_metadata_2023-12-20T21:43:29/validation_2023-12-20T21:51:11.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#3378/vernacular_metadata_2023-12-20T20:48:35/mapped_metadata_2023-12-20T21:43:29/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/3696/vernacular_metadata_2023-12-20T20:48:50/mapped_metadata_2023-12-20T21:43:31/validation_2023-12-20T21:51:12.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#3696/vernacular_metadata_2023-12-20T20:48:50/mapped_metadata_2023-12-20T21:43:31/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/5656/vernacular_metadata_2023-12-20T20:49:07/mapped_metadata_2023-12-20T21:43:33/validation_2023-12-20T21:51:13.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#5656/vernacular_metadata_2023-12-20T20:49:07/mapped_metadata_2023-12-20T21:43:33/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/8918/vernacular_metadata_2023-12-20T20:49:18/mapped_metadata_2023-12-20T21:43:34/validation_2023-12-20T21:51:17.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#8918/vernacular_metadata_2023-12-20T20:49:18/mapped_metadata_2023-12-20T21:43:34/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/9757/vernacular_metadata_2023-12-20T20:50:33/mapped_metadata_2023-12-20T21:43:41/validation_2023-12-20T21:51:33.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#9757/vernacular_metadata_2023-12-20T20:50:33/mapped_metadata_2023-12-20T21:43:41/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/10503/vernacular_metadata_2023-12-20T20:55:12/mapped_metadata_2023-12-20T21:44:07/validation_2023-12-20T21:51:36.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#10503/vernacular_metadata_2023-12-20T20:55:12/mapped_metadata_2023-12-20T21:44:07/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27202/vernacular_metadata_2023-12-20T20:55:59/mapped_metadata_2023-12-20T21:44:12/validation_2023-12-20T21:52:37.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#27202/vernacular_metadata_2023-12-20T20:55:59/mapped_metadata_2023-12-20T21:44:12/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27275/vernacular_metadata_2023-12-20T21:11:25/mapped_metadata_2023-12-20T21:45:37/validation_2023-12-20T21:52:41.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#27275/vernacular_metadata_2023-12-20T21:11:25/mapped_metadata_2023-12-20T21:45:37/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27306/vernacular_metadata_2023-12-20T21:12:41/mapped_metadata_2023-12-20T21:45:44/validation_2023-12-20T21:52:57.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#27306/vernacular_metadata_2023-12-20T21:12:41/mapped_metadata_2023-12-20T21:45:44/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27307/vernacular_metadata_2023-12-20T21:17:25/mapped_metadata_2023-12-20T21:46:11/validation_2023-12-20T21:53:04.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#27307/vernacular_metadata_2023-12-20T21:17:25/mapped_metadata_2023-12-20T21:46:11/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27548/vernacular_metadata_2023-12-20T21:19:17/mapped_metadata_2023-12-20T21:46:21/validation_2023-12-20T21:53:18.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#27548/vernacular_metadata_2023-12-20T21:19:17/mapped_metadata_2023-12-20T21:46:21/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/27685/vernacular_metadata_2023-12-20T21:23:21/mapped_metadata_2023-12-20T21:46:45/validation_2023-12-20T21:53:20.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#27685/vernacular_metadata_2023-12-20T21:23:21/mapped_metadata_2023-12-20T21:46:45/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28047/vernacular_metadata_2023-12-20T21:23:54/mapped_metadata_2023-12-20T21:46:48/validation_2023-12-20T21:53:21.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#28047/vernacular_metadata_2023-12-20T21:23:54/mapped_metadata_2023-12-20T21:46:48/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28048/vernacular_metadata_2023-12-20T21:24:14/mapped_metadata_2023-12-20T21:46:50/validation_2023-12-20T21:53:30.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#28048/vernacular_metadata_2023-12-20T21:24:14/mapped_metadata_2023-12-20T21:46:50/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28049/vernacular_metadata_2023-12-20T21:26:48/mapped_metadata_2023-12-20T21:47:04/validation_2023-12-20T21:53:31.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#28049/vernacular_metadata_2023-12-20T21:26:48/mapped_metadata_2023-12-20T21:47:04/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28050/vernacular_metadata_2023-12-20T21:27:08/mapped_metadata_2023-12-20T21:47:06/validation_2023-12-20T21:54:12.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#28050/vernacular_metadata_2023-12-20T21:27:08/mapped_metadata_2023-12-20T21:47:06/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28052/vernacular_metadata_2023-12-20T21:37:56/mapped_metadata_2023-12-20T21:48:04/validation_2023-12-20T21:54:14.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#28052/vernacular_metadata_2023-12-20T21:37:56/mapped_metadata_2023-12-20T21:48:04/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28053/vernacular_metadata_2023-12-20T21:38:14/mapped_metadata_2023-12-20T21:48:06/validation_2023-12-20T21:54:15.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#28053/vernacular_metadata_2023-12-20T21:38:14/mapped_metadata_2023-12-20T21:48:06/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28054/vernacular_metadata_2023-12-20T21:38:23/mapped_metadata_2023-12-20T21:48:07/validation_2023-12-20T21:54:16.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#28054/vernacular_metadata_2023-12-20T21:38:23/mapped_metadata_2023-12-20T21:48:07/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28055/vernacular_metadata_2023-12-20T21:38:30/mapped_metadata_2023-12-20T21:48:08/validation_2023-12-20T21:54:16.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#28055/vernacular_metadata_2023-12-20T21:38:30/mapped_metadata_2023-12-20T21:48:08/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28056/vernacular_metadata_2023-12-20T21:38:36/mapped_metadata_2023-12-20T21:48:09/validation_2023-12-20T21:54:18.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#28056/vernacular_metadata_2023-12-20T21:38:36/mapped_metadata_2023-12-20T21:48:09/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28057/vernacular_metadata_2023-12-20T21:39:05/mapped_metadata_2023-12-20T21:48:12/validation_2023-12-20T21:54:19.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#28057/vernacular_metadata_2023-12-20T21:39:05/mapped_metadata_2023-12-20T21:48:12/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28058/vernacular_metadata_2023-12-20T21:39:21/mapped_metadata_2023-12-20T21:48:13/validation_2023-12-20T21:54:19.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#28058/vernacular_metadata_2023-12-20T21:39:21/mapped_metadata_2023-12-20T21:48:13/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28059/vernacular_metadata_2023-12-20T21:39:21/mapped_metadata_2023-12-20T21:48:13/validation_2023-12-20T21:54:21.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#28059/vernacular_metadata_2023-12-20T21:39:21/mapped_metadata_2023-12-20T21:48:13/data/

[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Download validation report at: https://rikolti-data.s3.amazonaws.com/28060/vernacular_metadata_2023-12-20T21:39:52/mapped_metadata_2023-12-20T21:48:16/validation_2023-12-20T21:54:23.csv
[2023-12-20, 21:54:23 UTC] {{logging_mixin.py:150}} INFO - Review collection data at: https://{bucket}.s3.us-west-2.amazonaws.com/index.html#28060/vernacular_metadata_2023-12-20T21:39:52/mapped_metadata_2023-12-20T21:48:16/data/
